pkg/persistence/queries.go

```diff
@@ -127,7 +171,12 @@ func claimKey(db *sql.DB, oneTimeCode string, appPublicKey []byte) ([]byte, erro
 		return nil, err
 	}
 
-	row := s.QueryRow(appPublicKey)
+	row = s.QueryRow(appPublicKey)
+
+	event := Event{Originator: originator, DeviceType: Server, Identifier: OTKClaimed, Count: 1, Date: time.Now()}
+	if err := saveEvent(db, event); err != nil {
+		log(nil, err).Warn(fmt.Sprintf("Unable to log event: %#v\n", event))
```

Comment:
Throughout the code there are a couple of instances that it is logging with a formatted string. This ends up looking something like this in the logs: 

```
Logging Event {Identifier:OTKGenerated DeviceType:Server Date:2020-09-21 14:43:49.165884 +0000 UTC m=+454.608032701 Count:1 Originator:thisisaverylongtoken}  component=db
INFO[0454] http response
```

This would cause use to have more of a mix of formats in the logs. The logger function already has a `.WithField(k,v)` function that we may want to leverage instead. This would make a conversion to a JSON based format easier.

Reply:
We may also want to remove `Originator:thisisaverylongtoken` from the logs and use the `source` instead ex: `TestProvince`

Reply:
> We may also want to remove `Originator:thisisaverylongtoken` from the logs and use the `source` instead ex: `TestProvince`

Oof good catch, we definitely don't want to be storing tokens in logs.

Reply:
Good idea we now log unknown tokens as `first char...last char`
Also using WithFields